server/handlers/mutations/get-subscription-url.js

```diff
@@ -0,0 +1,50 @@
+import "isomorphic-fetch";
```

Comment:
why do we need anything about app subscription? it's a free app, remove

Reply:
Removed it